CMakeLists.txt

```diff
@@ -14,10 +19,10 @@ else ()
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -O2 -g -fpermissive")
 endif ()
 
-if (NOT EMSCRIPTEN)
-    set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
-    set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
-endif ()
+#if (NOT EMSCRIPTEN)
+#    set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+#    set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+#endif ()
```

Comment:
Remove commented out code. I will push a commit fixing the paths in the workflow file instead of overwriting the output directories.


```suggestion
```